source/declarations.tex

```diff
@@ -6774,9 +6774,9 @@
 that is a function template whose first template parameter
 is a non-type parameter,
 the initializer is
-\tcode{\exposidnc{e}.get<i>()}. Otherwise, the initializer is \tcode{get<i>(\exposid{e})},
+\tcode{\exposidnc{e}.std::get<i>()}. Otherwise, the initializer is \tcode{std::get<i>(\exposid{e})},
 where \tcode{get} undergoes argument-dependent lookup\iref{basic.lookup.argdep}.
-In either case, \tcode{get<i>} is interpreted as a \grammarterm{template-id}.
+In either case, \tcode{std::get<i>} is interpreted as a \grammarterm{template-id}.
```

Comment:
I don't think changing the language classifies as editorial.

Reply:
Yes, the core language does consider `get` an ADL-based customization point for structured bindings.